crawl-ref/source/directn.cc

```diff
@@ -3215,8 +3215,11 @@ string feature_description_at(const coord_def& where, bool covering,
 
     if (covering && you.see_cell(where))
     {
-        if (feat_is_tree(grid) && env.forest_awoken_until)
+        if (feat_is_tree(grid) && env.forest_awoken_until){
```

Comment:
```suggestion
        if (feat_is_tree(grid) && env.forest_awoken_until)
        {
```
Crawl style is braces on their own line. I'm in the process of merging a bunch of pull requests and will fix that, but it is a little speed bump.